src/senaite/core/tests/doctests/AnalysisTurnaroundTimeForZeroValues.rst

```diff
@@ -0,0 +1,182 @@
+    Analysis Turnaround Time For Zero Values
+------------------------
+
+Running this test from the buildout directory::
+
+    bin/test test_textual_doctests -t AnalysisTurnaroundTimeForZeroValues
+
+
+Test Setup
+..........
+
+Needed Imports:
+
+    >>> from bika.lims import api
+    >>> from bika.lims.utils.analysisrequest import create_analysisrequest
+    >>> from bika.lims.workflow import doActionFor
+    >>> from DateTime import DateTime
+    >>> from datetime import timedelta
+    >>> from plone.app.testing import TEST_USER_ID
+    >>> from plone.app.testing import setRoles
+    >>> from senaite.core.api.dtime import to_DT, to_dt
+
+Functional Helpers:
+
+    >>> def change_receive_date(ar, days):
+    ...     prev_date = ar.getDateReceived()
+    ...     ar.Schema().getField('DateReceived').set(ar, prev_date + days)
+    ...     for analysis in ar.getAnalyses(full_objects=True):
+    ...         an_created = analysis.created()
+    ...         analysis.getField('creation_date').set(analysis, an_created + days)
+
+    >>> def compute_due_date(analysis):
+    ...     start = to_dt(analysis.getStartProcessDate())
+    ...     tat = api.to_minutes(**analysis.getMaxTimeAllowed())
+    ...     due_date = start + timedelta(minutes=tat)
+    ...     return to_DT(due_date)
+
+    >>> def compute_duration(date_from, date_to):
+    ...     return (date_to - date_from) * 24 * 60
+
+Variables:
+
+    >>> portal = self.portal
+    >>> request = self.request
+    >>> setup = portal.setup
+    >>> bikasetup = portal.bika_setup
+
+We need to create some basic objects for the test:
+
+    >>> setRoles(portal, TEST_USER_ID, ['LabManager',])
+    >>> date_now = DateTime().strftime("%Y-%m-%d")
+    >>> date_future = (DateTime() + 5).strftime("%Y-%m-%d")
+    >>> client = api.create(portal.clients, "Client", Name="Happy Hills", ClientID="HH", MemberDiscountApplies=True)
+    >>> contact = api.create(client, "Contact", Firstname="Rita", Lastname="Mohale")
+    >>> sampletype = api.create(bikasetup.bika_sampletypes, "SampleType", title="Water", Prefix="W")
```

Comment:
Note test is [failing now](https://github.com/senaite/senaite.core/actions/runs/10302605876/job/28691381570?pr=2569#step:7:247) because of #2584 . Use `setup.sampletypes` instead of `bikasetup.bika_sampletypes`